lua/pac3/editor/server/wear.lua

```diff
@@ -232,26 +226,28 @@ function pace.SubmitPart(data, filter)
 	end
 
 	if not data.server_only then
-		if data.owner:IsValid() then
-			data.player_uid = pac.Hash(data.owner)
+		if owner:IsValid() then
+			data.player_uid = pac.Hash(owner)
 		end
 
-		local players = filter or players
+		players = filter or players
```

Comment:
local players?

Reply:
It's already localized up here: https://github.com/CFC-Servers/pac3/blob/fix/rework-submitpart/lua/pac3/editor/server/wear.lua#L204

Should be fine, right?

Reply:
oh right